### DISCUSSION TITLE
Conferma, per una Notifica di Trascrizione, ricevuta dallo stesso Comune che ha generato l'evento.

### DISCUSSION THREAD

**Discussion by gcuci** [italia/ansc#825] [Supporto servizi cooperativi] [answered]
Buongiorno,

dai nostri test in pre-produzione, si evince, che successivamente alla validazione di un evento, ANSC genera una Notifica di tipo Trascrizione, per ognuno degli intestatari dell'evento, per il Comune di Residenza di questi ultimi. Se questo è digitale.

Se l'intestatario è residente nel Comune che ha generato l'evento, la Notifica viene comunque inviata allo stesso Comune.

Deve essere effettuata, anche in questo caso, la Conferma della Notifica, da parte dell'USC? 

In attesa di riscontro,

cordiali saluti.

**Comment by mttfranci** [comment, marked as answer]
Salve, in seguito ai tavoli di analisi, è stato deciso che anche quando il comune di formazione coincide con quello di residenza è necessaria la conferma dell' USC.

Saluti

Assistenza ANSC
